js/directives/lucca/user-picker.js

```diff
@@ -113,6 +123,8 @@
 		var selectedUsersCount = 0;
 		// Only used for asynchronous pagination
 		var timeout = {}; // object that handles timeouts - timeout.count will store the id of the timeout related to the count query
+		// List of properties that will be fetched in case of homonyms
+		var props;
```

Comment:
you dont need this variable to be accessible by all the methods defined in this controller. I'd rather you define it line ~370 and pass it to methods needing it (`getHomonymsPropertiesAsync(homonyms, props)` for example)